### DISCUSSION TITLE
Can't find entities: need IDs

### DISCUSSION THREAD

**Discussion by Startrail82** [cgiesche/streamdeck-homeassistant#248] [Help and Support] [answered]
Hi!

I'm trying to integrate this plugin into my home management. Everything seems clear to me, but I'm bugged by the fact that I can only see entity names and not entity IDs. I have 1000+ entities in my home (don't ask...), so naturally, there are a lot of double named items. I really need to be able to set the entity ID (type it, select it, ...) and not scroll in a seemingly infinite list of entity names which are, as said, not distinguishable from each other.

What's the magic trick to make this happen?

**Comment by cgiesche** [comment, marked as answer]
Hi!

That sounds like a great idea. Even with unique names, scrolling through the list is not very comfortable.

I created an issue for the feature: #250

Preview:
![Animation](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/a4f06ba2-3087-460a-b007-fcfc201c0ec1)

**Comment by Startrail82** [comment]
Cool! 👑
For bonus points: show the entity ID, just as a verification?

**Comment by cgiesche** [comment]
I added the entity-id as a tooltip because I would have to make bigger changes to the selection box to display additional information. I will come back and refine this feature later.

**Comment by cgiesche** [comment]
![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/5123180/e0657d98-becd-4f0a-8cb4-44d2c9031095)

**Reply by Startrail82** [reply]
I've installed 2.4.5, but it doesn't really work. I don't get the dialog you demonstrate here.

![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/68431491/41cc52ba-f313-4f5f-9e49-cef9ca828cf5)
